unit_tests/RecordUnitTests.d

```diff
@@ -122,3 +122,167 @@ struct Foo
 D");
 
 }
+
+// Translate immediately declared array variable in global scope.
+unittest
+{
+    assertTranslates(
+q"C
+
+struct Bar {
+const char *foo;
+const char *oof;
+} baz[64];
+
+C",
+q"D
+extern (C):
+
+struct Bar
+{
+    const(char)* foo;
+    const(char)* oof;
+}
+
+extern __gshared Bar[64] baz;
+D");
+
+}
+
+// Do not put an extra newline after array declaration.
+unittest
+{
+    assertTranslates(q"C
+typedef int Baz;
+
+struct Foo {
+    Baz data[32];
+    char len;
+};
+C",
+q"D
+extern (C):
+
+struct Foo
+{
+    int[32] data;
```

Comment:
> Exactly. If the alias is reduced you can get different results depending on which platform the binding is generated on.

Ok, you convinced me. However I still don't like that only the standard types would be reduced. I think there should be a way for user to specify which aliases he want to reduce and to what type.

I mean time I can change it to reduce only types from `stdint.h` and from linux headers. Maybe with two CLI switches `reduce-stdint-aliases` and `reduce-__sxx-types`.